src/index.js

```diff
@@ -0,0 +1,78 @@
+import './style.css';
+import '@fortawesome/fontawesome-free/css/all.css';
+
+const taskList = document.querySelector('.task-list');
+
+class TaskStorage {
+  data = []
```

Comment:
For this milestone, kindly populate your data variable with a list of objects and populate your page with them.

<img width="1048" alt="image" src="https://github.com/ulises2607/ToDo-List/assets/27709832/2752a44a-7da2-404d-b32b-c4c1cff5e6d7">

Keep the add functionality you currently have as you'll need it in the milestones to come.